00-programming-fundamentals/08-intro-to-automated-tests.md

```diff
@@ -60,24 +66,273 @@ true
 __Question: Why bother?__
 In the above example, the code in `exponate_test.rb` are _automated tests_. We can run that file whenever we want to ensure that `exponate` is still performing reasonably. When our requirements change, we can feel secure in changing our code because we have some _tests_ to let us know if we've broken something.
 
-### Assertions
-An _assertion_ is the part of the _automated test suite_ that tells us whether our code did what we expected it to. Usually, _assertions_ are special methods that will evaluate whether an expression is `true`. If it is, the _assertion_ is true, the test passes (green test), otherwise it fails (red test).
 
-### Exercise: Write an `assert` method
-Let's build our own `assert` method in the `exponate_test.rb` file. Here's what it should look like:
+## Now Comes Minitest
+
+Writing your own tests with puts is wonderful, but it would be handy to have a standard way that developers can use to write tests on their code, a way that other developers understand.  The maintainers of the Ruby language have adopted a testing library called [Minitest](http://docs.seattlerb.org/minitest/) as the default standard for testing in Ruby & later Rails.  For the remainder of your time using Ruby at Ada, we will be using Minitest to write unit-tests for your code.  There are other testing frameworks, notably [RSpec](http://rspec.info/	), but the key concepts are similar.  
```

Comment:
For the last sentence some thing like: "RSpec is another very common testing framework used along with Ruby and Rails. We won't be using it here at Ada but it's good to know about when you're browsing the internet for testing help."